src/main/java/inf112/skeleton/app/object/Deck.java

```diff
@@ -0,0 +1,55 @@
+package inf112.skeleton.app.object;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Deck {
+    ArrayList<Card> cards;
+    ArrayList<Card> discard;
+
+    public Deck() {
+        cards = new ArrayList<Card>();
+        discard = new ArrayList<Card>();
+
+        init();
+    }
```

Comment:
Shouldn't the cards be shuffled at the initialization of the deck?
If I am not mistaken the way it is written now it will draw the cards in the order that they are added in the `init` method.
Again, not gamebreaking or merge blocking, but it is a nice to have feature at later point.